### ISSUE TITLE
I can not get the pesq result that reported in this work

### ISSUE THREAD

**Issue by ndisci** [Qinwen-Hu/dparn#11]
Hi,
I used your pretrained model and 48k sampling rate VCTK data as test set . But the test data's pesq is about 2.2931. According to paper the model was better than other models. So what wrongs in this result?  How can I get the result correctly? 
Pytorch verison is 1.10.1.
Thanks.

**Comment by Qinwen-Hu** [comment]
Hi，
I tested the same pretrained model on the VCTK-DEMAND dataset and got **2.98** for the  wide-band PESQ score (It is better than the score in the article for it's trained on a larger dataset). 
And I'm sure there's no problem with my PESQ calculating tool, as the PESQ score for VCTK-DEMAND noisy dataset is verified to be consistent with what's provided in the article(**1.97**).
Are you sure you calculated the PESQ scores correctly? It should be measured on the down-sampled 16k signals.
If there's no problem on the PESQ calculating tool, maybe you can send me your code for inferencing? I've never tried it in pytorch 1.10.1 and I'm not so sure what caused the difference.
Qinwen, Hu.

**Comment by ndisci** [comment]
I tested the data set for wide-band pesq score,too. So, which tool did you use ? 
First I enhanced noisy file and downsampled it to 16k signal. After that I calculated pesq score using 16k downsampled clean signal and enhanced signal. So what wrongs in these steps ?

**Comment by Qinwen-Hu** [comment]
I used the Matlab implementation for PESQ and STOI.
It seems that there's nothing wrong in your steps. If the score you calculated for the noisy data is consistent with that in the paper, then there shouldn't be any problems. 
I've uploaded some 48k VCTK-DEMAND demo to the repository as a reference, and the enhanced clips are processed by the provided pretrained model. If they are too different from your processed results, then maybe it's due to the pytorch version? I'll figure it out when I get enough time (Being a little bit busy recently.  :D)

**Comment by ndisci** [comment]
I am using this database : https://datashare.ed.ac.uk/handle/10283/2791
The scores aren't the same. (2.98 vs. 2.29)
It can be due to PyTorch version, you're right. What is your cuda version for torch 1.7.1 ? I can downgrade the torch version.
Thanks for your time.

**Comment by Qinwen-Hu** [comment]
My CUDA version is 10.1. &nbsp; &nbsp;
Thanks for your attention to the work.&nbsp;





------------------ Original ------------------
From: ndisci ***@***.***&gt;
Date: Tue,Sep 6,2022 8:07 PM
To: Qinwen-Hu/dparn ***@***.***&gt;
Cc: Qinwen Hu ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [Qinwen-Hu/dparn] I can not get the pesq result that reported in this work (Issue #11)





 
I am using this database : https://datashare.ed.ac.uk/handle/10283/2791
 The scores aren't the same. (2.98 vs. 2.29)
 It can be due to PyTorch version, you're right. What is your cuda version for torch 1.7.1 ? I can downgrade the torch version.
 Thanks for your time.

**Comment by ndisci** [comment]
You're welcome. I got 3.00 pesq score with PyTorch 1.7.1+cu11.0. So It could be because of torch version(for example torch.irfft works differently from old torch.fft.irfft)

**Comment by Qinwen-Hu** [comment]
Glad that it workd :D
